internal/shifts/recordshift.py

```diff
@@ -46,7 +46,7 @@ def process(self, command):
             if command.is_lift:
                 coord = command.getPadCoord()
                 if coord == (0, 0):
-                    beat.toggleMetronome()
```

Comment:
Oh unless your idea is to abstract all this functionality within the BeatMgr class, as it looks a bit sparse right now?

Reply:
Ooooo this is exciting I'll try this out in a bit! Thanks for the fix!

Reply:
Yep this looks like it works!